src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,25 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+interface Props {
+  todo: Todo
+}
+
+export const TodoInfo = ({ todo }: Props) => {
+  return (
+    <article
+      data-id={todo.id}
+      className={classNames(
```

Comment:
wrong usage of `classNames` - check it [here](https://www.npmjs.com/package/classnames)